unattended-upgrade

```diff
@@ -1664,7 +1664,8 @@ def main(options, rootdir=""):
     if not options.dry_run:
         log_content = get_dpkg_log_content(logfile_dpkg, install_start_time)
         send_summary_mail(
```

Comment:
Given that we use `pkg_install_sucess and pkg_install manual` here and below I think it makes sense to have something like: `successful_run = pkg_install_success and not pkg_install_manual`.